frontend/js/src/common/brainzplayer/YoutubePlayer.tsx

```diff
@@ -357,7 +358,8 @@ export default class YoutubePlayer extends React.Component<YoutubePlayerProps>
   };
 
   render() {
-    const { show } = this.props;
+    const { show, volume } = this.props;
+    this.youtubePlayer?.setVolume(volume ?? 100);
```

Comment:
Same as for the others, this shouldn't be called in the render function but rather after setting up `this.youtubePlayer`, probably in the `onReady` function